app/containers/Products/ProductRowItemContainer.tsx

```diff
@@ -209,15 +179,13 @@ export const ProductRowItemComponent: React.FunctionComponent<Props> = ({
 
     newLinks.forEach(async (id) => {
       if (!previousLinks.includes(id)) {
-        const response = await createLinkedProduct(id);
-        console.log(response);
+        await createLinkedProduct(id);
       }
     });
 
     previousLinks.forEach(async (id, index) => {
       if (!newLinks.includes(id)) {
-        const response = await removeLinkedProduct(linkData[index]);
-        console.log(response);
+        await removeLinkedProduct(linkData[index]);
```

Comment:
is there a way we can bundle up these in a single mutation? Or is it too much work since there aren't many links updated at the same time?

Reply:
It's unlikely there are many links updated at one time. Looking in the database the most links one product has is 4, so probably unnecessary to put the work in to bundle them. Investigation batch mutations could be a tech-debt card, but one we probably don't need to tackle until we have a need for it.

Reply:
Mutations are already batched in a single HTTP request, so it the gain would be marginal (if a single mutation can even update more than one record?)